bower.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
Fair point, but it's still working. I would not drop it yet, we can always discuss it in a separate issue and remove at any moment. This is not so heavy to maintain as the other things solved by this PR.